### PR TITLE
Documentation,manifests: Document exposing reporting API outside of cluster

### PR DESCRIPTION
**Pull request by chancez**
Adds documentation on how to expose the reporting API so that you can query reports outside of the cluster.
Also documents how to enable the openshift oauth proxy to protect the API when exposing it.

### CODE REVIEW

**Comment by chancez** [comment]
With #491 merged, I'm trying to decide if this could be pulled into a separate document. I feel like this plus the prometheus URL config _could_ go into a new `configuring-reporting-operator.md` document which we link from both the `metering-config.md` and from `common-configuration.md` since this is such a common piece. 

This is making me want auto-generated table of contents for markdown.

File: manifests/metering-config/expose-route.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        #   testuser:{SHA}y/2sYAj5yrQIN4TL0YdPdmGNKpc=
+        #
+        # change REPLACEME to the output of your htpasswd command
+        htpasswdData: |
+          REPLACEME
+
+        # cookieSeed is used to protect the cookie created if accessing the API
+        # via your browser.
+        #
+        # generate a 32 character random string using `pwgen 32`
```
Comment (pruan-rht) [inline manifests/metering-config/expose-route.yaml]: FYI, `pwgen` is not included in the default openshift installation as well as Mac/Windows.  I have a better luck using `openssl rand -base64 32` . Perhaps we can use that suggestion instead

Comment (chancez) [reply manifests/metering-config/expose-route.yaml]: Feel free to open a PR

Comment (pruan-rht) [reply manifests/metering-config/expose-route.yaml]: ok, opened #497

**Pull request merged**